types/micromatch/index.d.ts

```diff
@@ -353,7 +353,7 @@ interface Micromatch {
      * //=> ['a.a', 'a.aa']
      * ```
      */
-    match(list: readonly string[], pattern: string, options?: micromatch.Options): string[];
+    match(list: readonly string[], pattern: string | string[], options?: micromatch.Options): string[];
```

Comment:
It should match the main function signature:
```suggestion
    match(list: readonly string[], pattern: string | readonly string[], options?: micromatch.Options): string[];
```